eventsapi/models.py

```diff
@@ -1,5 +1,35 @@
-from pydantic import BaseModel
+from typing import Annotated, Literal, Union
+from pydantic import Field, BaseModel
+from uuid import UUID
 
 
 class DetailMessage(BaseModel):
     detail: str
+
+
+class BaseEvent(BaseModel):
+    course_id: int
+    impression_id: UUID
+    source_uri: str
+    timestamp: int
+
+
+class ContentLoadSuccessV1Event(BaseEvent):
+    eventname: Literal['content_load_success_v1']
```

Comment:
I see where you're going with that - events are always records of things happening, and so our descriptions could be verbs as well. I like that. I also think having them past tense is good practice, since they are kinda like receipts with timestamps. @P-Gill97 Are you ok with that?